TransShiftMex/Audapter.cpp

```diff
@@ -1356,6 +1361,41 @@ void *Audapter::setGetParam(bool bSet,
                     p.timeDomainPitchShiftAlgorithm,
                     p.sr, p.frameLen, p.timeDomainPitchShiftSchedule));
         }
+
+		if (bRemakeFilter) {
+			if (p.sr < 24000) {
+				const dtype t_srfilt_a[nCoeffsSRFilt] = { 1.000000000000000000000000, -4.137689759094149300000000, 11.417342955970334000000000, -21.230389508442666000000000,
```

Comment:
Can you refactor the code so that these coefficients are located right next to the coefficients of the default sampling rate?

Reply:
I'd be happy to refactor, but sorry, I don't totally understand what you want refactored. Do you want the old and new filters to not be in separate if/else statements? I can instantiate and define both filters right next to each other, then just use an if/else to call `downSampFilter.setCoeff` and `upSampFilter.setCoeff` on the proper filter. Is that what you mean?